Lib/test/test_argparse.py

```diff
@@ -1938,7 +1938,23 @@ def test_required_subparsers_no_destination_error(self):
         parser = ErrorRaisingArgumentParser()
         subparsers = parser.add_subparsers()
         subparsers.add_parser('run')
-        self.assertArgumentParserError(parser.parse_args, ())
+        with self.assertRaises(ArgumentParserError) as excinfo:
+            parser.parse_args(())
+        self.assertRegex(
+            excinfo.exception.stderr,
+            'error: the following arguments are required: {run}\n$'
+        )
+
+    def test_wrong_argument_subparsers_no_destination_error(self):
+        parser = ErrorRaisingArgumentParser()
+        subparsers = parser.add_subparsers()
+        subparsers.add_parser('foo')
+        with self.assertRaises(ArgumentParserError) as excinfo:
+            parser.parse_args(('bar',))
+        self.assertRegex(
+            excinfo.exception.stderr,
+            r"error: argument {foo}: invalid choice: 'bar' \(choose from 'foo'\)\n$"
```

Comment:
I expected different markup: `argument foo: invalid choice "bar": choose from {"foo"}`

Reply:
(i.e. match what argparse does for a regular choices arg/option)

Reply:
I think this is correct as is and it becomes more obvious with more than one choice -- let me update the test to demonstrate that.